README.md

```diff
@@ -30,11 +33,54 @@ and this to your crate root:
 ```rust
 extern crate rand;
 
-// example usage:
-use rand::{Rng, thread_rng};
-let x: f64 = thread_rng().gen();
+use rand::prelude::*;
+
+// basic usage with random():
+let x: u8 = rand::random();
```

Comment:
don't need `rand::` prefix with the prelude

Reply:
If we advertise `random` so prominently, shouldn't we also mention the caveats?

Reply:
Which do you have in mind to mention?

Reply:
That calling `random` often is not recommended, because it is more efficient to reuse the RNG.

Reply:
Not here, no: (1) premature optimisation (actually there's not a big overhead anyway); (2) it's in the `random` doc and keeps this simple.